test/common/testlib.py

```diff
@@ -654,6 +662,112 @@ def snapshot(self, title, label=None):
             attach(filename)
             print("Wrote HTML dump to " + filename)
 
+    def assert_pixels(self, selector, key, ignore=[]):
+        """Compare the given element with its reference"""
+
+        if Image and self.pixels_label and self.cdp and self.cdp.valid:
+            rect = self.call_js_func('ph_element_clip', selector)
```

Comment:
Nitpick: Invert the condition, return, and avoid the giant indentation of essentially the whole function.

Reply:
Good point, done!